From 7fa08989fb69f8959c7398327f9b7cbd656a0e97 Mon Sep 17 00:00:00 2001
From: Iain Russell <Iain.Russell@ecmwf.int>
Date: Fri, 31 Mar 2023 17:48:46 +0100
Subject: [PATCH] Require C++17 compiler

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4bcfcd15..360c6546 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -25,7 +25,7 @@ set(INSTALL_INCLUDE_DIR include/magics CACHE PATH "Magics installation directory
 
 include(ecbuild_system NO_POLICY_SCOPE)
 
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_C_STANDARD 99)
 
-- 
2.32.0 (Apple Git-132)

