From b8d1f45b5fc8de3d512f596fe14b63965b54a372 Mon Sep 17 00:00:00 2001
From: Iain Russell <Iain.Russell@ecmwf.int>
Date: Thu, 22 Aug 2019 13:24:07 +0100
Subject: [PATCH] BUILD: do not export TPL paths in build directory [MAGP-1203]

---
 CMakeLists.txt | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d6a804a6..2f4b2b2a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -239,6 +239,22 @@ set( MAGICS_LIBRARIES MagPlus )
 
 foreach( _tpl ${MAGICS_TPLS} )
     string( TOUPPER ${_tpl} TPL )
+
+    if( REPLACE_TPL_ABSOLUTE_PATHS )
+        # replace TPL absolute paths with their library names
+        # this helps make Magics relocatable
+        set( _TMP "" )
+
+        foreach( _lib ${${TPL}_LIBRARIES} )
+            get_filename_component( _lib_name ${_lib} NAME_WE )
+            string( REGEX REPLACE "^lib" "" _name ${_lib_name} )
+            list( APPEND _TMP "-l${_name}" )
+        endforeach()
+
+        set( ${TPL}_LIBRARIES ${_TMP} )
+        set( _TMP "" )
+    endif()
+
     list( APPEND MAGICS_EXTRA_DEFINITIONS ${${TPL}_DEFINITIONS}  )
     list( APPEND MAGICS_EXTRA_INCLUDE_DIRS ${${TPL}_INCLUDE_DIRS} )
     string(REGEX REPLACE "[ ]+$" "" ${TPL}_LIBRARIES "${${TPL}_LIBRARIES}") # strips leading whitespaces
-- 
2.13.7

