From 46af1b43500e9607466dcf812161baa3426d9a01 Mon Sep 17 00:00:00 2001
From: Iain Russell <Iain.Russell@ecmwf.int>
Date: Mon, 2 Sep 2019 11:07:13 +0100
Subject: [PATCH] BUILD: do not export TPL paths in build directory [MAGP-1203]

---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2f4b2b2a..47634a8a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -250,6 +250,9 @@ foreach( _tpl ${MAGICS_TPLS} )
             string( REGEX REPLACE "^lib" "" _name ${_lib_name} )
             list( APPEND _TMP "-l${_name}" )
         endforeach()
+        list(REMOVE_DUPLICATES  _TMP)
+        list(REMOVE_ITEM _TMP -ldebug)
+        list(REMOVE_ITEM _TMP -loptimized)
 
         set( ${TPL}_LIBRARIES ${_TMP} )
         set( _TMP "" )
-- 
2.13.7

