From dfcd04153fd86f7d0cca9c51f08a51a93e234fc5 Mon Sep 17 00:00:00 2001
From: Iain Russell <Iain.Russell@ecmwf.int>
Date: Mon, 2 Sep 2019 12:23:59 +0100
Subject: [PATCH] BUILD: do not export TPL paths in build directory [MAGP-1203]

---
 CMakeLists.txt | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 47634a8a..9fb5a21c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -248,7 +248,11 @@ foreach( _tpl ${MAGICS_TPLS} )
         foreach( _lib ${${TPL}_LIBRARIES} )
             get_filename_component( _lib_name ${_lib} NAME_WE )
             string( REGEX REPLACE "^lib" "" _name ${_lib_name} )
-            list( APPEND _TMP "-l${_name}" )
+            if( NOT ${_name} STREQUAL ${_lib_name} )
+              list( APPEND _TMP "-l${_name}" )
+            else()
+              list( APPEND _TMP "${_lib}" )
+          endif()
         endforeach()
         list(REMOVE_DUPLICATES  _TMP)
         list(REMOVE_ITEM _TMP -ldebug)
@@ -266,7 +270,6 @@ foreach( _tpl ${MAGICS_TPLS} )
     ecbuild_info("${TPL}_LIBRARIES [${${TPL}_LIBRARIES}]") # add this to confirm that indeed it was cleaned up
 endforeach()
 
-
 if( MAGICS_QT )
   list( APPEND MAGICS_EXTRA_INCLUDE_DIRS  ${Qt5Widgets_INCLUDE_DIR} )
   list( APPEND MAGICS_EXTRA_LIBRARIES     ${Qt5Widgets_LIBRARIES} )
-- 
2.13.7

