From 24ebe5c814e32141363b04a9383ea16efd9de487 Mon Sep 17 00:00:00 2001
From: Iain Russell <Iain.Russell@ecmwf.int>
Date: Mon, 2 Sep 2019 12:49:49 +0100
Subject: [PATCH] BUILD: do not export TPL paths in build directory [MAGP-1203]

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9fb5a21c..fab21f12 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -255,8 +255,8 @@ foreach( _tpl ${MAGICS_TPLS} )
           endif()
         endforeach()
         list(REMOVE_DUPLICATES  _TMP)
-        list(REMOVE_ITEM _TMP -ldebug)
-        list(REMOVE_ITEM _TMP -loptimized)
+        list(REMOVE_ITEM _TMP debug)
+        list(REMOVE_ITEM _TMP optimized)
 
         set( ${TPL}_LIBRARIES ${_TMP} )
         set( _TMP "" )
-- 
2.13.7

