From 137bda00e467252c2e193ecd8a94cb84d72c80da Mon Sep 17 00:00:00 2001
From: Iain Russell <Iain.Russell@ecmwf.int>
Date: Wed, 4 Sep 2019 10:06:16 +0100
Subject: [PATCH] BUILD: do not export TPL paths in build directory [MAGP-1203]

---
 CMakeLists.txt | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index fab21f12..d5c9ceee 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -246,17 +246,23 @@ foreach( _tpl ${MAGICS_TPLS} )
         set( _TMP "" )
 
         foreach( _lib ${${TPL}_LIBRARIES} )
+            message(STATUS "inspecting ${_lib}")
             get_filename_component( _lib_name ${_lib} NAME_WE )
+            message(STATUS "  _lib_name: ${_lib_name}")
             string( REGEX REPLACE "^lib" "" _name ${_lib_name} )
+            message(STATUS "  after replace: ${_name}")
             if( NOT ${_name} STREQUAL ${_lib_name} )
+              message(STATUS "  adding new: -l${_name}")
               list( APPEND _TMP "-l${_name}" )
             else()
               list( APPEND _TMP "${_lib}" )
+              message(STATUS "  adding old: ${_lib}")
           endif()
         endforeach()
         list(REMOVE_DUPLICATES  _TMP)
         list(REMOVE_ITEM _TMP debug)
         list(REMOVE_ITEM _TMP optimized)
+        message(STATUS "After all that: ${_TMP}")
 
         set( ${TPL}_LIBRARIES ${_TMP} )
         set( _TMP "" )
-- 
2.13.7

