From 1785da4d665edd1f8b2c412d494ef0eb603f31cd Mon Sep 17 00:00:00 2001
From: Iain Russell <Iain.Russell@ecmwf.int>
Date: Mon, 7 Oct 2019 14:39:59 +0100
Subject: [PATCH] Fix for Windows build

---
 src/web/MagConfig.cc | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/src/web/MagConfig.cc b/src/web/MagConfig.cc
index 7b8f3c1d..717078b6 100644
--- a/src/web/MagConfig.cc
+++ b/src/web/MagConfig.cc
@@ -230,10 +230,8 @@ void StyleLibrary::init() {
                     else
                         MagConfigHandler(path + "/" + current_, *this);
                 }
-                while (!_findnext(handle, &fileinfo))
-                    ;
-                _findclose(handle);
-            }
+            } while (!_findnext(handle, &fileinfo))
+            _findclose(handle);
         }
 #endif
     }
-- 
2.13.7

