From 1da8288b16d77ebc04152580ff8f53e0c43bbb3a Mon Sep 17 00:00:00 2001
From: Iain Russell <Iain.Russell@ecmwf.int>
Date: Mon, 7 Oct 2019 14:54:47 +0100
Subject: [PATCH] Fix for Windows build

---
 src/web/MagConfig.cc | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/web/MagConfig.cc b/src/web/MagConfig.cc
index 1c8e2683..4afa1a65 100644
--- a/src/web/MagConfig.cc
+++ b/src/web/MagConfig.cc
@@ -230,10 +230,10 @@ void StyleLibrary::init() {
                     else
                         MagConfigHandler(path + "/" + current_, *this);
                 }
-                while (!_findnext(handle, &fileinfo))
-                    ;
-                _findclose(handle);
-            }
+            } while (!_findnext(handle, &fileinfo))
+
+            _findclose(handle);
+        }
 #endif
     }
 }
-- 
2.13.7

